<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnRoomBooking">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Room Booking</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRoomCheckout">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Check-Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCheckAvailability">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Check Availability</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bntTransaction">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Transactions</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>421</width>
      <height>361</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
